core/src/mindustry/world/blocks/storage/CoreBlock.java

```diff
@@ -48,7 +48,7 @@ public CoreBlock(String name){
         update = true;
         hasItems = true;
         priority = TargetPriority.core;
-        flags = EnumSet.of(BlockFlag.core, BlockFlag.unitModifier);
+        flags = EnumSet.of(BlockFlag.core, BlockFlag.unitModifier, BlockFlag.storage);
```

Comment:
This may mess up logic. Consider looping through every core and connected container instead.

Reply:
will do tomorrow 👍